.gitignore

```diff
@@ -6,4 +6,5 @@ __pycache__
 bin/
 lib/
 cmake-build-*/
-.idea/
\ No newline at end of file
+.idea/
+sandbox.cpp
```

Comment:
Intentionally leaving `sandbox.cpp` here because I'll probably do the same thing in the future.